ocs_ci/ocs/vector_utils.py

```
@@ -0,0 +1,448 @@
+    Example:
+        vectors = generate_test_vectors_with_metadata(
+            dimension=3,
+            num_vectors=100
+        )
+
+    """
+    genres = ["scifi", "family", "drama", "action", "comedy"]
+    vectors = []
+
+    for i in range(num_vectors):
+        vector_data = [random.uniform(0.0, 2.0) for _ in range(dimension)]
```

Ruff (0.15.13)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)